Document the TMPDIR environment variable for better discovery (#308)
swiftlang/swiftly: Documentation/SwiftlyDocs.docc/automated-install.md

- term `SWIFTLY_HOME_DIR`: The location of the swiftly configuration files, and environment scripts
- term `SWIFTLY_BIN_DIR`: The location of the swiftly binary and toolchain symbolic links (for example swift, swiftc, and so on)

> - term `TMPDIR`: The temporary directory to be used by swiftly for temporary files, such as downloads that can be very large until they are cleaned up.

Reviewer: ```suggestion
- term `TMPDIR`: The temporary directory swiftly uses to hold large files, such as downloads, until it cleans them up.
```
